custom_components/tuya_local/devices/windmill_airconditioner.yaml

```diff
@@ -0,0 +1,53 @@
+name: Windmill Air Conditioner
+products:
+  - id: ebd7ef1c98deabe827aukn
+    name: Windmill Air Conditioner
+primary_entity:
+  entity: climate
+  dps:
+    - id: 1
+      name: hvac_mode
+      type: boolean
+      mapping:
+        - dps_val: false
+          value: "off"
+        - dps_val: true
+          constraint: mode
+          conditions:
+            - dps_val: wind
+              value: fan_only
+            - dps_val: cool
+              value: cool
+            - dps_val: eco
+              value: auto
```

Comment:
I understand that there is an "eco" preset, and it may make sense to map that to this instead. But for this device, these options (wind, cool, eco) are mutually exclusive, so IDK if it makes sense to have them be separate properties that could be set at the same time. Plus, all the "eco" mode does is automatically turn the cooling off (but keep the fan on) when the temp is at the target temp, so "auto" seems appropriate.

Reply:
This is good. Better to handle it along with the other hvac_mode if it is mutually exclusive with those on the device itself.